src/ssh.c

```diff
@@ -734,6 +766,12 @@ int wolfSSH_connect(WOLFSSH* ssh)
             /* no break */
 
         case CONNECT_SERVER_USERAUTH_REQUEST_DONE:
+            #ifdef WOLFSSH_AGENT
+                if (ssh->agentEnabled) {
+                    ssh->agent = wolfSSH_AGENT_new(ssh->ctx->heap);
```

Comment:
should there be a NULL return check here

Reply:
Fixed.